src/databricks/labs/ucx/assessment/queries/database_summary.sql

```diff
@@ -0,0 +1,39 @@
+-- viz type=table, name=Database Summary, columns=database,tables,views,dbfs root,delta tables,upgrade
+-- widget title=Database Summary, col=0, row=6, size_x=3, size_y=8
+SELECT `database`,
+       SUM(IS_TABLE) AS TABLES,
```

Comment:
nit: column names have to be lowercase, otherwise it's a bit confusing to read

Reply:
Darn, this was the formatter's doing.